README.md

```diff
@@ -23,104 +23,332 @@
     </tbody>
 </table>
 
+## Overview
+
+Firebase Hosting integrates with popular modern Web frameworks including Angular and Next.js. Using Firebase Hosting and
+Cloud Functions for Firebase with these frameworks, you can develop apps and microservices in your preferred framework
+environment, and then deploy them in a managed, secure server environment. Support during this early preview includes
+the following functionality:
+
+* Deploy Web apps comprised of static web content
+* Deploy Web apps that use pre-rendering / Static Site Generation (SSG)
+* Deploy Web apps that use server-side Rendering (SSR)—full server rendering on demand
+
+Firebase provides this functionality through the Firebase CLI. While initializing Hosting on the command line, you
+provide information about your new or existing Web project, and the CLI sets up the right resources for your chosen Web
+framework.
+
 ## Status
 
 ![Status: Experimental](https://img.shields.io/badge/Status-Experimental-blue)
 
-This repository is maintained by Googlers but is not a supported Firebase product. Issues here are answered by maintainers and other community members on GitHub on a best-effort basis.
+This repository is maintained by Googlers but is not a supported Firebase product. Issues here are answered by
+maintainers and other community members on GitHub on a best-effort basis.
 
 # Enable framework-awareness
 
 This is an experimental addon to the Firebase CLI to add web framework support, to enable it call the following:
 
 ```shell
-firebase --open-sesame frameworkawareness
+firebase experiments:enable webframeworks
 ```
 
-# Deploy Next.js
+# Prerequisites
+
+- Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
+
+### Serve locally
```

Comment:
Is this meant to come under Prerequisites, or should we think about ordering this differently?